projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -491,7 +491,7 @@ protected function get_episode_image_url( SimplePie\Item $episode, $itunes_ns =
 	 */
 	protected function get_audio_enclosure( SimplePie\Item $episode ) {
 		foreach ( (array) $episode->get_enclosures() as $enclosure ) {
-			if ( str_starts_with( $enclosure->type, 'audio/' ) ) {
+			if ( str_starts_with( $enclosure->type ?? '', 'audio/' ) ) {
```

Comment:
I find myself waffling a bit between the above and this:
```suggestion
			if ( isset( $enclosure->type ) && str_starts_with( $enclosure->type, 'audio/' ) ) {
```
They have the same result, and the `??` version is more concise. I'm unsure whether I find the `isset` version more readable or not.

Reply:
I waffled between those, along with an `is_string()` check and even an `is_object()` check. 😅 I'm going to keep it concise for now.